packages/api/src/platforms/vtex/resolvers/product.ts

```diff
@@ -157,5 +157,15 @@ export const StoreProduct: Record<string, Resolver<Root>> & {
   },
   releaseDate: ({ isVariantOf: { releaseDate } }) => releaseDate ?? '',
   advertisement: ({ isVariantOf: { advertisement } }) => advertisement,
-  rating: (item) => item.rating,
+  rating: (item) => ({
+    average: item.rating.average,
+    totalCount: item.rating.totalCount,
+    distribution: {
+      starsOne: item.rating.distribution?.[1] ?? 0,
+      starsTwo: item.rating.distribution?.[2] ?? 0,
+      starsThree: item.rating.distribution?.[3] ?? 0,
+      starsFour: item.rating.distribution?.[4] ?? 0,
+      starsFive: item.rating.distribution?.[5] ?? 0,
+    },
+  }),
```

Comment:
suggestion: use destructuring to make it more direct, what do you think?
```suggestion
  rating: ({ rating: { average, totalCount, distribution } }) => ({
    average,
    totalCount,
    distribution: {
      starsOne: distribution?.[1] ?? 0,
      starsTwo: distribution?.[2] ?? 0,
      starsThree: distribution?.[3] ?? 0,
      starsFour: distribution?.[4] ?? 0,
      starsFive: distribution?.[5] ?? 0,
    },
  }),
```

Reply:
done